src/helix-ui/styles/navigation.less

```diff
@@ -14,13 +14,11 @@
   font-size: 0.875rem; /* ~14px */
   letter-spacing: 0.75px;
   padding-top: 2rem;
-  width: 15rem; /* ~240px */
 
   hx-disclosure {
     color: inherit;
     display: block;
     padding: @vpad @rpad @vpad @l1-indent;
-    text-transform: uppercase;
```

Comment:
Refactored to `> hx-disclosure` to ensure it only applies to the top level items.